config/environment.rb

```diff
@@ -34,7 +34,7 @@
     migrations 'db/migrations'
     schema     'db/schema.sql'
 
-    logger "log/development.log", level: :debug
+    logger "log/#{Hanami.env}.log", level: :debug
```

Comment:
I'm not sure that I want to see test logs and production logs (heroku have some issues about it). Could you explain why we need this changes?